ui_framework/src/components/table/_index.scss

```diff
@@ -0,0 +1,12 @@
+$kuiTableCellPadding: $kuiSize;
+$kuiTableFontSize: 14px;
```

Comment:
And really, because it's used only once, I think you can just put the value directly in table.scss